README.md

```diff
@@ -34,13 +34,13 @@ To learn about contributing to Legend, see the [CONTRIBUTING.md](CONTRIBUTING.md
 ## Contribute to FINOS projects that use Legend as a data modeling tool
 Several FINOS projects are leveraging Legend for data modeling efforts under the [FINOS Financial Object Special Interest Group](https://github.com/finos/financial-objects) umbrella. Get involved by [joining the next Financial Objects SIG meeting](https://github.com/finos/financial-objects/issues?q=label%3Ameeting) or by emailing the Financial Objects community at [fo@finos.org](mailto:fo@finos.org).
 
-### Shared Public Instance of Legend Studio
+### Shared public instance of Legend Studio
```

Comment:
This should say "instance of Legend" and not "Legend Studio". FINOS hosts the entire installation.
And in the future there will be other components like Maven repositories etc that people can/will use.